client/src/components/TagField.scss

```diff
@@ -1,5 +1,5 @@
-@import "~react-select/dist/react-select.css";
+@import "~react-select/scss/default";
 
-.ss-tag-field .Select--multi .Select-value {
+.Select--multi .Select-value {
```

Comment:
This change is a little concerning - looks like it'd be then applying to any react selects in the CMS...?

Reply:
Yeah. I figured that'd be okay - I only did this because I had to work around this requirement with entwine and then again with storybook. I'll just change it back and workaround in storybook though.